src/HomePage/HomePage.js

```diff
@@ -11,7 +12,10 @@ class HomePage extends Component {
         <div className="main">
           <div className="fronPageBannerContainer">
             <h1 className="mainTitle">Monsrud Open</h1>
-            <h4>Monsrud Open 2020 skjer 15. august 2020</h4>
+            <video width="800" height="auto" controls>
```

Comment:
veldig bra! Om du ønsker kan man legge ting ting her for å autoavspille. Men det er jo også litt irriterende.